paddle/test_recognize_digits_conv.py

```diff
@@ -60,6 +60,31 @@
 
 exe.run(framework.default_startup_program())
 
+
+def eval_test():
+    test_reader = paddle.batch(
+        paddle.dataset.mnist.test(), batch_size=BATCH_SIZE)
+    accuracy.reset(exe)
+    for batch_id, data in enumerate(test_reader()):
+        img_data = np.array(map(lambda x: x[0].reshape([1, 28, 28]),
+                                data)).astype(DTYPE)
+        y_data = np.array(map(lambda x: x[1], data)).astype("int64")
+        y_data = y_data.reshape([len(y_data), 1])
+
+        tensor_img = core.LoDTensor()
+        tensor_y = core.LoDTensor()
+        tensor_img.set(img_data, place)
+        tensor_y.set(y_data, place)
```

Comment:
The latest code does not need to call LoDTensor() here, please to refer the latest code in develop branch.

Reply:
In our latest branch, we can 
1. use LoDTensor(), call executor.run(return_numpy=False)
2. use lists of numpy, other things use the default configuration.